### ISSUE TITLE
[Raid] Halion - The Twilight Destroyer- Twilight Cutter bugged

### ISSUE THREAD

**Issue by squalllh1989** [WoWManiaUK/Redemption#4424]
**Links:**https://www.wow-mania.com/armory/?spell=77846

**What is Happening:** Currently the Twilight Cutter kill people that not touch it.
You can see on this video: https://www.youtube.com/watch?v=F5IxcTTXCE4&t=6s
Quick mark: at 3:42 in the video, you can see a warrior with shadowmoure is dpsing, at 3:43-3:44 he disappear - is death and body send to physical realm, and you can see that he is no where near the cutter to touch it.

**What Should happen:** The cutter should only do damage and kill to people who touch it, not to people who are not.

**Comment by Aristo14** [comment]
Most likely lag related, using your video kinda obviously shows this in my opinion. 

![Screenshot_2](https://user-images.githubusercontent.com/38132718/79079495-c4535b00-7d0f-11ea-9fab-3da08be547bd.png)

On this cutter all went well, look at the distance the people are from the cutter. 

![Screenshot_3](https://user-images.githubusercontent.com/38132718/79079516-e220c000-7d0f-11ea-877b-6dbe3f04da30.png)

Here is where it went wrong, the time mark you pointed out. 

The only difference is that the first picture people stacked really tight, and the 2nd picture the warrior might have been to much in the front.  The other people dying seemed fine but probably died due to lag. 

Long story short, if the cutters where bugged, you would have a lot of deaths on the first picture. 

(leaving it open for testers to pitch in)

**Comment by Oganku** [comment]
Delay can make others move almost a full second later on your screen than they actually do on theirs, it can lead to wonky looking things sometimes. Probably what happened?

**Comment by Keader** [comment]
I cant reproduce.
Without reproduce steps is impossible to fix :/
